### DISCUSSION TITLE
what is this building for

### DISCUSSION THREAD

**Discussion by JokerJostar** [ayamir/nvimdots#1106] [Q&A] [answered]
i had stuck here about 1 hour
![IMG_20231223_025841](https://github.com/ayamir/nvimdots/assets/94276943/fb6c4733-5902-43fc-85f1-745906b50a32)
![IMG_20231223_025837](https://github.com/ayamir/nvimdots/assets/94276943/7639e657-9246-4012-b137-6055ec40954c)
![IMG_20231223_025841](https://github.com/ayamir/nvimdots/assets/94276943/6893ad3f-8c3e-4b53-8f24-e9e7defb0423)

**Comment by ayamir** [comment, marked as answer]
No need to install ruby dependency, it's **optional** actually.
